menhir-recover/main.ml

```diff
@@ -67,6 +75,62 @@ let () =
       );
   end
 
+let () =
+  if !graph then
+    let open G in
+    Lr1.iter (fun lr ->
+      let file_name = !graph_folder ^ string_of_int (Lr1.to_int lr) ^ ".html" in
+      let channel = open_out file_name in
+      let file = Format.formatter_of_out_channel channel in
+      Format.fprintf file "<!DOCTYPE html>\n<html>\n<body>\n";
+      Format.fprintf file "<h1>%d</h1>\n" @@ Lr1.to_int lr;
+      Format.fprintf file "<table>\n";
+
+      Format.fprintf file "<tr><td style=\"vertical-align:top\">\n";
+      Format.fprintf file "<table>\n<th>LR0</th>\n";
+      Format.fprintf file "<tr><th>Incoming</th></tr>\n";
```

Comment:
Also, I think we can use here it simple combinator or function like `print_table` for example.
Ideally, we can use a simple html-DSL library for it (e. g. Tyxml), but maybe now it isn't vital.

Reply:
done